modular_skyrat/modules/welding_sparks/code/weldingtool.dm

```diff
@@ -0,0 +1,23 @@
+/**
+ * Modularity is a neccessary evil.
+ *
+ * It makes everything far harder, yet it is the only way to achieve salvation.
+ *
+ * Without it, we would be nothing.
+ *
+ * These are the side effects of our choice to achieve salvation.
+ */
+
+/obj/item
+	/// An image effect for displaying a welding effect for items that support it.
+	var/image/welding_sparks
+
+/obj/item/Initialize(mapload)
+	. = ..()
+	if(tool_behaviour == TOOL_WELDER) // If we act like a welder, set up some spark effects.
```

Comment:
early return

Reply:
No, not on initialize.

Reply:
'. = ..()' makes the default value on a 'return' with no arguments whatever '..()' returned, so no harm in doing that

Reply:
it makes little difference but it's best to follow guidelines